src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -1215,17 +1215,21 @@ class DeviceArgument(cli.Argument):
     )
     DEVICE_NAME = cli.ArgumentProperties(
         key='device_name',
-        flags=('--name',),
+        flags=('-n', '--name'),
```

Comment:
```suggestion
        flags=('--name', '-n'),
```

Shorthand flag is expected to come after the long version. Then both command line and markdown docs refer to the descriptive name before the more cryptic shorthand. (Note the changes in autogenerated docs).